.github/workflows/stale.yml

```diff
@@ -1,22 +0,0 @@
-name: "Handle stale PRs and issues"
```

Comment:
Are you sure you don't want to keep it for PRs?

Reply:
Hm, has it proven useful in your opinion?

Reply:
I suppose not recently since we've been diligent in keeping our open PRs relevant 🤔